.ci/run-tests

```diff
@@ -5,7 +5,7 @@
 # Default environment variables
 export TEST_SUITE="${TEST_SUITE:=oss}"
 export PYTHON_VERSION="${PYTHON_VERSION:=3.9}"
-export PANDAS_VERSION=${PANDAS_VERSION-1.5.0}
+export PANDAS_VERSION=${PANDAS_VERSION-2.1.4}
```

Comment:
```suggestion

```

Reply:
for starters, try to find out where this environment variable `PANDAS_VERSION` is being used and try to remove the usage.